pipeline/src/transformers/article.ts

```diff
@@ -101,7 +101,11 @@ export const transformArticle = (
 
   const format = isFilledLinkToDocumentWithData(data.format)
     ? transformLabelType(data.format)
-    : undefined;
+    : ({
+        type: "ArticleFormat",
+        id: "W7TfJRAAAJ1D0eLK",
+        label: "Article",
+      } as ArticleFormat);
```

Comment:
Minor: when we're hard-coding formats like this, maybe put them in a top-level directory similar to `data` in the front-end repo? So we can see all the hard-coded data in one place.